phalcon/http/response.zep

```diff
@@ -423,6 +424,23 @@ class Response implements ResponseInterface, InjectionAwareInterface
 	}
 
 	/**
+	 * Sets the response content-length
+	 *
+	 *<code>
+	 *	$response->setContentLength(2048);
+	 *</code>
+	 *
+	 * @param int contentLength
+	 * @return \Phalcon\Http\Response
```

Comment:
We automatically generate the documentation based on the code. Zephir already knows how to do PHPDoc.
![123](https://cloud.githubusercontent.com/assets/1256298/16939222/485766aa-4d88-11e6-8078-a7e7aa5dd52e.png)